### DISCUSSION TITLE
Патч v4.13.1

### DISCUSSION THREAD

**Discussion by rovius** [LacunaHub/.github#37] [Новости]
- **Плагины**:
  * Добавлена возможность локализации созданных плагинов ([подробнее](<https://github.com/LacunaHub/LacunaPluginTemplate/>))
- **Панель управления**:
  * Исправлена ошибка, из-за которой невозможно было установить локализацию, отличную от русской, английской и украинской
  * Исправлена ошибка, из-за которой опция поддержки через Top.gg не перенаправляла на страницу для голосования
  * Размеры диалогов и области редактора изображений теперь зависят от размеров окна 
  * Обновлены иконки для категорий настроек
- **Расширение для Discord AutoMod**:
  * Удалено поле "Игнорируемые каналы" для триггера проверки имен пользователей

**Comment by MAR10ds** [comment]
Пожалуйста добавьте временные предупреждения 🙏.
